### PR TITLE
default options before all other options

### PR DESCRIPTION
**Pull request by vuillaut**
The main changes concerns the slurm options in `SbatchLstMCStage` and config files.
These options are now handled as dictionnaries,  allowing more flexibility and clarity.
The "general" options (account, error or output files...) are still set on their own but can be overwritten by specific slurm options in some stages.
Default options are easily overwritten too, but the user does not need to specify the full extent of them as it was the case before.

- [x] fix slurm_options
- [x] modify paths_configs generation accordingly

### CODE REVIEW

**Comment by vuillaut** [comment]
@garciagenrique ready for review    
I will do a full test on the cluster tomorrow.

**Review by garciagenrique — APPROVED** [review]
Don't wanna block the PR for the formatting, but please change it

File: lstmcpipe/stages/mc_dl2_to_sensitivity.py [hunk 1]
```diff
@@ -36,44 +30,31 @@ def batch_dl2_to_sensitivity(
...
-
     for paths in dict_paths:
-        job_logs, jobid = dl2_to_sensitivity(
-            paths["input"],
-            paths["output"],
-            batch_configuration=batch_config,
-            wait_jobs_dl1_dl2=job_ids_from_dl1_dl2,
-            slurm_options=paths.get("slurm_options", None),
-        )
+        job_logs, jobid = dl2_to_sensitivity(paths["input"], paths["output"],
```
[the remark this answers is not in the document]
Comment (vuillaut) [reply lstmcpipe/stages/mc_dl2_to_sensitivity.py]: I am not sure what should I do, I have run the pre-commit on these files.
?

**Comment by vuillaut** [comment]
The allsky test worked well, merging.

**Pull request merged**